### DISCUSSION TITLE
productAttributeValues high load time

### DISCUSSION THREAD

**Discussion by zssvaidar** [atrocore/atrocore#628] [Q&A] [answered]
On page product/{id}
-> request ```https://pim.meloman.kz/api/v1/Product/63e9cec5b22b78cc0/productAttributeValues```
takes long time load even only 16 attributes(mostly empty)
![image](https://user-images.githubusercontent.com/30953478/220825857-923f47c0-7ff4-4908-9dcc-1df2e92a0f9f.png)

does all the data from "typeValue" are neccessary to load and send client ?
![image](https://user-images.githubusercontent.com/30953478/220826024-4259420f-cfb6-4ff6-8830-8786a5eb7779.png)

**Comment by rratsun** [comment]
The size of response is too big. Ok, we created an internal issue and will refactor this problem soon. I will write the version of core where this problem will be solved.

**Comment by rratsun** [comment, marked as answer]
Fixed at PIM 1.7.13
